lib/test-reader/mocha-test-parser.js

```diff
@@ -51,6 +52,14 @@ module.exports = class MochaTestParser extends EventEmitter {
         this._passthroughFileEvents();
     }
 
+    _setTestExecTimeout() {
```

Comment:
you just need to use `testExecTimeout` at https://github.com/gemini-testing/hermione/blob/master/lib/worker/runner/test-runner/execution-thread.js#L49 without mocha timeouts